payment/controllers.go

```diff
@@ -483,7 +482,7 @@ func DeleteOrderCreds(service *Service) handlers.AppHandler {
 			)
 		}
 
-		err := service.Datastore.DeleteOrderCreds(*orderID.UUID())
+		err := service.Datastore.DeleteOrderCreds(*orderID.UUID(), false)
```

Comment:
should this not be passed as a query param?

Reply:
resolved: https://github.com/brave-intl/bat-go/pull/816/commits/3282e3de323c11fc713d3726e72d812f8abee27b